### ISSUE TITLE
Android App crashes on app start up on version 1.73.114-t09706 on Android 9-10

### ISSUE THREAD

**Issue by charlieporth1** [tailscale/tailscale#13448]
### What is the issue?
Android App crashes on app start up on version 1.73.114-t09706
Has crashed on 2 Android 10 phones

Clearing storage/cache does not fix this.
Nor uninstalling and reinstalling
### Steps to reproduce

Needed: Android 10
Android 11+ works fine
Update app. 
Should crash

### Are there any recent changes that introduced the issue?

Beta version 1.73.114-t09706

### OS

Android

### OS version

Android 10

### Tailscale version

1.73.114-t09706

### Bug report

N/A crashing causes no ability to get this

**Comment by amadeumagalhaes** [comment]
Same problem here : Sony Bravia A80K

**Comment by agottardo** [comment]
We are aware of the issue and have identified the root cause. Stay tuned for updates.

**Comment by agottardo** [comment]
For posterity:
```
pid: 7719, tid: 7764, name: DefaultDispatch >>> com.tailscale.ipn << 
uid: 10145
signal 31 (SIGSYS), code 1 (SYS_SECCOMP), fault addr
Cause: seccomp prevented call to disallowed arm64 system call 434
```

**Comment by bradfitz** [comment]
Details in #13452

**Comment by charlieporth1** [comment]
Recommended action for those who are a unaware is to downgrade the installed app to 1.72 using an APK file and ADB

**Comment by agottardo** [comment]
@charlieporth1 or alternatively, just open the Play Store and leave the beta channel?

**Comment by agottardo** [comment]
This is fixed in 1.75.3 available in the beta channel.
